template.yaml

```diff
@@ -39,23 +42,23 @@ Resources:
             Schedule: rate(1 hour)
             Description: Once an hour
             Enabled: True
+
   FollowerCount:
     Type: AWS::DynamoDB::Table
     Properties:
       AttributeDefinitions:
-        - AttributeName: EpochTime
-          AttributeType: "N"
         - AttributeName: twitter_handle
           AttributeType: "S"
+        - AttributeName: EpochTime
+          AttributeType: "N"
       KeySchema:
-        - AttributeName: "EpochTime"
-          KeyType: "HASH"
         - AttributeName: "twitter_handle"
+          KeyType: "HASH"
+        - AttributeName: "EpochTime"
           KeyType: "RANGE"
-      ProvisionedThroughput: 
+      ProvisionedThroughput:
           ReadCapacityUnits: 1
           WriteCapacityUnits: 1
-      TableName: FollowerCount
```

Comment:
This is a good idea, but it will create a new table with you deploy this over an existing stack

Reply:
changing the schema would as well.  You would need to backup the current table before doing it, then you could load the historical data back in.  You would have 59 minutes...  Unless you do a new stack side by side.  :)